### PR TITLE
auth_base: save auth result to Results

### PR DESCRIPTION
**Pull request by msimerson**
so that auth puts this into ES output:
````
      "auth/auth_vpopmaild": {
        "pass": [
          "PLAIN"
        ]
      },
````

drive-by:
* wrap long lines
* remove trailing whitespace

### CODE REVIEW

File: plugins/auth/auth_base.js [hunk 1]
```diff
@@ -100,15 +103,19 @@ exports.check_user = function (next, connection, credentials, method) {
...
         connection.notes.auth_login_userlogin = null;
         connection.notes.auth_login_asked_login = false;
 
         var delay = Math.pow(2, connection.notes.auth_fails - 1);
-        if (plugin.timeout && delay >= plugin.timeout) { delay = plugin.timeout - 1; }
-        connection.lognotice(plugin, 'delaying response for ' + delay + ' seconds');
+        if (plugin.timeout && delay >= plugin.timeout) {
+            delay = plugin.timeout - 1;
+        }
+        connection.lognotice(plugin, 'delaying for ' + delay + ' seconds');
```
Comment (celesteking) [inline plugins/auth/auth_base.js]: whose idea is to log with notice and not debug level? It's very unimportant.

Comment (smfreegard) [inline plugins/auth/auth_base.js]: Submit a pull request to change it then.

Comment (celesteking) [inline plugins/auth/auth_base.js]: https://github.com/baudehlo/Haraka/commit/f385b44043f1d97949ec937d05abb320e78f6da9
i can't create a pull request, don't know why.

Comment (msimerson) [inline plugins/auth/auth_base.js]: We could be more helpful if you tell us how you tried creating the PR and at which point [in the contributing process](https://github.com/baudehlo/Haraka/wiki/Contributing) you got stuck.

**Pull request merged**